packages/graphback-cli/src/templates/modelTemplates.ts

```diff
@@ -0,0 +1,97 @@
+import { writeFileSync } from 'fs';
+import { prompt as ask } from 'inquirer'
+import ora from 'ora'
+import { Model } from './templateMetadata'
+
+export const defaultModel: string = `# type Note {
```

Comment:
Would it be possible to have them as separate files. each as separate graphql file. We can read them directly using fs

Reply:
We would have to copy them in build then, Examples aren't that large anyways